src/endpoint/s3/s3_utils.js

```diff
@@ -271,6 +271,17 @@ function set_response_object_md(res, object_md) {
     if (storage_class !== STORAGE_CLASS_STANDARD) {
         res.setHeader('x-amz-storage-class', storage_class);
     }
+    if (object_md.restore_status) {
+        const restore = [`ongoing-request="${object_md.restore_status.ongoing}"`];
+        if (!object_md.restore_status.ongoing && object_md.restore_status.expiry_time) {
+            // Expiry time is in UTC format
+            const expiry_date = new Date(object_md.restore_status.expiry_time).toUTCString();
```

Comment:
but what is the date format of the xattr `user.noobaa.restore.expiry` ?
is it a number? worth noting it here in the comment at least.
in nb.d.ts I see this is a Date... So unclear.

Reply:
Mentioned the format in the xattr comment (All the data types are mentioned in the comments).